sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpConnectionScope.cs

```diff
@@ -1104,7 +1150,9 @@ private static AmqpSettings CreateAmpqSettings(Version amqpVersion)
             return new WebSocketTransportSettings
             {
                 Uri = uriBuilder.Uri,
-                Proxy = proxy ?? (default)
+                Proxy = proxy ?? (default),
```

Comment:
is `default` different than null?

Reply:
No, in this case it is not.   I think that was a remnant of when we had a concrete instance, but I don't recall.   It's weird syntax, wrapped in parenthesis.   I removed the coalesce.